src/config/api.js

```diff
@@ -1,6 +1,7 @@
 import { create } from 'apisauce';
+import { CamelcaseSerializer, SnakecaseSerializer } from 'cerealizr';
 
-const baseURL = 'http://wolox.com';
+const baseURL = 'https://books-training-rails.herokuapp.com/api/v1';
```

Comment:
this should be on a `.env` as `REACT_APP_API_URL=https://books-training-rails.herokuapp.com/api/v1`